docs/source/configuration/refine_admin.rst

```diff
@@ -0,0 +1,35 @@
+Refine Admin
+============
+
+Some administrative functionality is built out using `Refine <https://refine.dev>`_. This is a separate service and requires a bit of setup to make it work with the MITxOnline app. 
+
+Accessing
+---------
+
+The Docker compose environment builds a ``refine`` container that runs a dev server available at port 8016. 
+
+Set Up
+------
+
+The Refine app uses OAuth2/OIDC for authentication with the core MITxOnline app. You will need to generate a shared private key for this authentication flow to work. 
+
+*Generating the Key:* The following command will generate the key:
+
+``openssl genrsa -out oidc.key 4096``
+
+The key then needs to be put into the ``.env`` file into a setting called ``OIDC_RSA_PRIVATE_KEY``. Remove linebreaks and replace with the string ``\n``. Both the Refine and Django apps will pick the key up from this setting.
+
+(The key generation process is from the `Django OAuth Toolkit <https://django-oauth-toolkit.readthedocs.io/en/latest/oidc.html#creating-rsa-private-key>`_ docs.)
+
+*Configuring MITxOnline:* You will need to add a new Application in the Django Oauth Toolkit section in Django Admin (``/admin/oauth2_provider/application/``). Navigate there and create a new Application. The settings you will need are:
+
+* **Client Id**: ``refine-local-client-id``
```

Comment:
Do we need to define client_id and client_secret somewhere in .env file ?

Reply:
<img width="684" alt="Screenshot 2022-05-25 at 2 51 42 PM" src="https://user-images.githubusercontent.com/7334669/170234982-2428c351-e8ae-4718-945e-03a3a0867bcd.png">


getting this while clicking on SignIn button.

Reply:
No, Client Secret is ignored (OIDC uses the key) and Client Id doesn't have an exposed configuration setting in the .env file right now so it's just that value.